Makefile

```diff
@@ -63,7 +63,7 @@ test: run-unit-tests
 lint: build-slim build-test
 	# Note: Make sure that you run `git config diff.noprefix false` in this repo
 	# See this issue for more details: https://github.com/golangci/golangci-lint/issues/948
-	golangci-lint run --enable-all --disable=godox,gochecknoglobals --max-same-issues=0 --max-issues-per-linter=0 --build-tags $(ALL_BUILD_TAGS) --new-from-rev=$$(git merge-base $$(cat .git/resource/base_sha 2>/dev/null || echo "master") HEAD) --modules-download-mode=$(MOD) --timeout=5m --exclude-use-default=false ./...
+	golangci-lint run --enable-all --disable=godox,gochecknoglobals,dupl --max-same-issues=0 --max-issues-per-linter=0 --build-tags $(ALL_BUILD_TAGS) --new-from-rev=$$(git merge-base $$(cat .git/resource/base_sha 2>/dev/null || echo "master") HEAD) --modules-download-mode=$(MOD) --timeout=5m --exclude-use-default=false ./...
```

Comment:
I wouldn't disable `dupl` linter. The code introduced in this PR is clearly a duplication. I'd rather use `nolint:` if we can't spend time on refactoring.

Reply:
this is no longer an issue so this was removed.